### PR TITLE
Fixes/hardcoded links

### PR DESCRIPTION
**Pull request by sturobson**
There were _several_ instances where we had hardcoded the `href` to either be empty `href=""` or contain a `#`. 

This PR intends to fix that by making all links use `.yml` data to populate. 

There's also a small fix to the `vf-pagination` component so that it's more a configurable template.

### CODE REVIEW

File: components/embl-logo/embl-logo.config.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ context:
   component-type: embl-element
   href: http://www.embl.de
```
Comment (khawkins98) [inline components/embl-logo/embl-logo.config.yml]: There's also an `href` in addition to `logo_href`, I'm guessing this accidental?

File: components/vf-pagination/vf-pagination.config.yml [hunk 2]
```diff
@@ -1,8 +1,92 @@
 title: Pagination
 label: Pagination
-status: alpha
-variants:
-  - name: default
-    hidden: true
+status: beta
```
Comment (khawkins98) [inline components/vf-pagination/vf-pagination.config.yml]: Pushed a commit to set the package.json to also be beta status.

File: components/embl-logo/embl-logo.config.yml (hunk 1 above)
Comment (sturobson) [reply components/embl-logo/embl-logo.config.yml]: this is 'technically' ok - as the `href` is the `.yml` not actually in the code as `href="#"`.

But this does bring up the lack of consistency with the key naming for this across components.

I think something like `component_link` or `component_href` would be better.

I'll refactor so they're all on the same hymn sheet.

Comment (khawkins98) [reply components/embl-logo/embl-logo.config.yml]: What I was getting at is you've added a `logo_href`, I'm guessing you just forgot to delete the old one? Or is it used somewhere I'm not seeing?

Comment (sturobson) [reply components/embl-logo/embl-logo.config.yml]: hmmm, now why wasn't that used before :sob:

I'll still go through and standardise the naming convention and do another push

**Comment by khawkins98** [comment]
Cool. Probably good to take the time on this one as once we push any change will make David's life hard.

**Pull request merged**